drivers/pds/api/accountRoleBinding.go

```diff
@@ -90,3 +89,20 @@ func (accountRoleBinding *AccountRoleBinding) AddUser(accountID string, email st
 	}
 	return nil
 }
+
+// RemoveUser function delete the user from given account.
+func (accountRoleBinding *AccountRoleBinding) RemoveUser(userID string) (*status.Response, error) {
+	client := accountRoleBinding.apiClient.AccountRoleBindingsApi
+	ctx, err := pdsutils.GetContext()
+	if err != nil {
+		log.Errorf("Error in getting context for api call: %v\n", err)
```

Comment:
Instead of logging and then returning, directly return nil, fmr.Errorf(....)

Reply:
Done